Objects/src/RecoElectron.cc

```diff
@@ -109,7 +101,7 @@ RecoElectron::etaSC() const
 Double_t
 RecoElectron::phiSC() const
```

Comment:
Are we actually using this function anywhere? If not, then I advocate for removing it.

Reply:
removed